chopsticks/tunnel.py

```diff
@@ -679,21 +679,25 @@ class SSHTunnel(SubprocessTunnel):
     :param host: The hostname to connect to, as would be specified on an
                  ``ssh`` command line.
     :param user: The username to connect as.
+    :param port: A custom port other than 22 to connect to.
```

Comment:
This is slightly misleading; if not given it defaults to whatever is in `ssh_config`, and only then defaults to 22.

A description such as "The TCP to connect to" would perhaps be sufficient, as the behaviour is similar to `user`, and we don't give details for that. Alternatively we could some details here about what happens when either of these parameters is None .